### DISCUSSION TITLE
Font Family Improvements: Enhance UI with Modern Fonts

### DISCUSSION THREAD

**Discussion by gtlYashParmar** [open-webui/open-webui#3469] [Ideas]
**Is your feature request related to a problem? Please describe.**
I'm often dissatisfied with the current font family used in our UI because it doesn't seem modern or visually appealing. The combination of Mona Sans and Arimo fonts can feel dated and doesn't provide a cohesive look across our application.

**Describe the solution you'd like**
I would like to see an improvement in the font family used in our UI, specifically replacing the current fonts with more modern and stylish options. I'm thinking of fonts like Inter, Google Sans, or other popular fonts. This change would enhance the overall user experience and give our application a fresher look.

**Describe alternatives you've considered**
I've considered using other font families such as Lexend, Outfit, Open Sans, or Montserrat, but I think Inter or Google Sans would be a better fit for our UI due to their clean and modern designs. 

**Additional context**
No additional context at this time, but I'd be happy to provide more information if needed.

**Comment by Samik-Pandit** [comment]
Spot on! I feel the same.

**Comment by gtlrohan** [comment]
I agree with you. The inter or outfit will work well for this.

**Comment by robertpiosik** [comment]
#3691

**Comment by robertpiosik** [comment]
Hi y'all. PR got merged and Open WebUI will default to Inter.

**Reply by gtlYashParmar** [reply]
LGTM! Thank you

**Comment by regs01** [comment]
Inter doesn't look good in Firefox. Has noticeable staircase (aliasing artifacts) effect even on high DPI screens.

**Comment by Iniquitatis** [comment]
Also, italics are quite hard to read. Fairly sure it's just a programmatically rendered oblique variant and it's far too skewed.

**UPDATE:** Nope, it's what Inter provides by default. The point still stands, though.
